src/esbuild.js

```diff
@@ -31,7 +31,7 @@ esbuild.build({
   outdir: '../dist',
   format: 'esm',
   minify: AppConstants.NODE_ENV !== 'dev',
-  sourcemap: AppConstants.NODE_ENV !== 'dev',
+  sourcemap: 'linked',
```

Comment:
The [docs are confusing here](https://esbuild.github.io/api/#sourcemap), but the code example suggests this value should be `true`, as originally coded.  I imagine `'linked'` might work as well, though I haven't tested.